qa/mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
```

Comment:
do we really need so many mypy.ini files? I though one file (`ceph.git/src/mypy.ini`) would be enough for everyone.

Reply:
With that one, I get `Found 2442 errors in 99 files (checked 194 source files)`. So this needs to be incrementally fixed imo. That's why I added a new file.